src/Analyzers/Core/Analyzers/xlf/AnalyzersResources.fr.xlf

```diff
@@ -227,6 +227,11 @@
         <target state="translated">Supprimer le cast inutile</target>
         <note />
       </trans-unit>
+      <trans-unit id="Remove_redundant_equality_with_true">
+        <source>Remove redundant equality with true</source>
+        <target state="new">Remove redundant equality with true</target>
```

Comment:
`Remove redundant equality check`

this will also work for the `!= false` case.